rules/manager.go

```diff
@@ -49,7 +45,7 @@ var (
 			Name:      "rule_evaluation_duration_seconds",
 			Help:      "The duration for a rule to execute.",
 		},
-		[]string{ruleTypeLabel},
+		[]string{"rule_type", "status"},
```

Comment:
We had this discussion a good time back, and all of these were removed from the code base. They're harder to work with, as it's not a simple division to get a failure ratio.

You should also be initilizing the time series values to 0.